docs/configuration/toml-file.md

```diff
@@ -10,6 +10,11 @@ environment = "testnet-1" # or "mainnet"
 [connections] # section for connections-related params
 server_addr = "127.0.0.1:1234"
 inbound_limit = 30
+outbound_limit = 27
```

Comment:
:see_no_evil:

Reply:
:rofl: I was thinking about documenting the fields and I didn't think that much about the actual values. I'll change it to be more reasonable